src/lua/bpf/bpf.lua

```diff
@@ -475,7 +475,7 @@ end
 
 local function LOAD(dst, src, off, vtype)
 	local base = V[src].const
-	assert(base.__dissector, 'NYI: load() on variable that doesnt have dissector')
+	assert(base.__dissector, 'NYI: load() on variable that doesn't have dissector')
```

Comment:
This breaks the build -- since the string is single quoted, adding a quote to "doesnt" breaks the quotes.

Reply:
I'll fix.